build/dockerfiles/KServe/entrypoint.sh

```diff
@@ -23,6 +23,13 @@ elif [ -z "$KIT_USER" ] && [ -z "$KIT_PASSWORD" ] &&  [ -n "$AWS_ROLE_ARN" ]; th
   AWS_ACCOUNT_ID=$(echo "$AWS_ROLE_ARN" | cut -d: -f5)
   echo "Logging into AWS ECR $AWS_ACCOUNT_ID.dkr.ecr.$AWS_ECR_REGION.amazonaws.com"
   aws ecr get-login-password --region $AWS_ECR_REGION | kit login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.$AWS_ECR_REGION.amazonaws.com
+elif [ -z "$KIT_USER" ] && [ -z "$KIT_PASSWORD" ] && [ "$GCP_WIF" == "1" ]; then
```

Comment:
Minor nit: I prefer this style of bash variable to be just the string "true" instead of "1". I feel it looks better in yaml in e.g. a k8s deploy.

```suggestion
elif [ -z "$KIT_USER" ] && [ -z "$KIT_PASSWORD" ] && [ "$GCP_WIF" == "true" ]; then
```